### ISSUE TITLE
Wrong version number for release/2.9

### ISSUE THREAD

**Issue by peiva-git** [PaddlePaddle/PaddleSeg#3584]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleSeg/issues)(包括open与closed)，没有发现相似的bug。I have searched the [open and closed issues](https://github.com/PaddlePaddle/PaddleSeg/issues) and found no similar bug report.

### Bug描述 Describe the Bug

The version number in `paddleseg.__init__.py` on branch release/2.9 is `0.0.0.dev` instead of `2.9.0`.

This part:
```python
from . import models, datasets, transforms, optimizers

__version__ = '0.0.0.dev0'
```

Should be changed to:
```python
from . import models, datasets, transforms, optimizers

__version__ = '2.9.0'
```

### 复现环境 Environment

- Linux
- PaddlePaddle: 2.5.1
- PaddleSeg: 2.9.0
- Python: 3.9.18
- CUDA: 10.2

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [X] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by shiyutang** [comment]
你好，欢迎提交PR修复

**Comment by peiva-git** [comment]
Closed in #3585
